src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.DeadlineCommand;
+import duke.command.DeleteCommand;
+import duke.command.EventCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.MarkCommand;
+import duke.command.ToDoCommand;
+import duke.command.UnmarkCommand;
+import duke.command.WelcomeCommand;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Class for parsing input string into commands to execute.
+ *
+ * @author dexter-sim
+ * @version 0.1
+ */
+public class Parser {
+    /**
+     * Returns a command based on the specified user input.
+     *
+     * @param userInput Represents the user input into the system.
+     * @return A command based on the user input.
+     * @throws DukeException If user input is an invalid string command.
+     */
+    public static Command parseInput(String userInput) throws DukeException {
+        String[] split = userInput.split(" ");
+        if (userInput.equals("welcome") && split.length == 1) {
```

Comment:
Do you think it will be neater to implement the conditionals as a switch case rather than multiple ifs statements?